src/cluster.py

```diff
@@ -535,27 +520,6 @@ def member_inactive(self) -> bool:
             "restarting",
         ]
 
-    @property
-    def member_replication_lag(self) -> str:
-        """Member replication lag."""
-        try:
-            for attempt in Retrying(stop=stop_after_delay(60), wait=wait_fixed(3)):
-                with attempt:
-                    cluster_status = requests.get(
-                        f"{self._patroni_url}/{PATRONI_CLUSTER_STATUS_ENDPOINT}",
-                        verify=self.verify,
-                        timeout=API_REQUEST_TIMEOUT,
-                        auth=self._patroni_auth,
-                    )
-        except RetryError:
-            return "unknown"
-
-        for member in cluster_status.json()["members"]:
-            if member["name"] == self.member_name:
-                return member.get("lag", "unknown")
-
-        return "unknown"
-
     @property
     def is_member_isolated(self) -> bool:
```

Comment:
There's another call to cluster status here, but need to verify how the response will look on an isolated unit.